src/lib/eina/eina_inline_lock_win32.x

```diff
@@ -0,0 +1,749 @@
+/* EINA - EFL data type library
+ * Copyright (C) 2020 Lucas Cavalcante de Sousa
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;
+ * if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EINA_INLINE_LOCK_WIN32_X_
+#define EINA_INLINE_LOCK_WIN32_X_
+
+#ifndef WIN32_LEAN_AND_MEAN
+# define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <Windows.h>
+#include <synchapi.h>
+#undef WIN32_LEAN_AND_MEAN
+
+#include "unimplemented.h"
+
```

Comment:
maybe remove this and others new line between include?

Reply:
I don't know. They were originally from `eina_inline_lock_posix.c`. Do you have any suggestions?